api/src/main/java/io/druid/data/input/impl/TimestampSpec.java

```diff
@@ -47,12 +47,12 @@
 
   private final String timestampColumn;
   private final String timestampFormat;
-  private final Function<Object, DateTime> timestampConverter;
   // this value should never be set for production data
   private final DateTime missingValue;
+  private final transient Function<Object, DateTime> timestampConverter;
```

Comment:
I thought transient just marks fields that shouldn't be serialized. But this class isn't Serializable, so what's the point?